components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/config/WebSubAdapterConfiguration.java

```diff
@@ -33,7 +33,9 @@ public class WebSubAdapterConfiguration {
 
     private static final String ADAPTER_ENABLED_CONFIG = "adapter.websubhub.enabled";
     private static final String ENCRYPTION_ENABLED_CONFIG = "adapter.websubhub.encryptionEnabled";
+    private static final String TOPIC_DELETION_DISABLED_CONFIG = "adapter.websubhub.topicDeletionDisabled";
```

Comment:
Pls link all issues of the feature developments included in this PR.

Reply:
Linked